.github/workflows/main.yml

```diff
@@ -30,3 +30,36 @@ jobs:
         run: python -m pip install -U tox
       - name: Run lint
         run: tox -elint
+  build:
+    runs-on: ubuntu-latest  # windows-latest | macos-latest
+    name: Test changed-files
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # OR "2" -> To retrieve the preceding commit.
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v24
+        with:
+          files: |
+            red_queen/games/applications/*.py
+            red_queen/games/mapping/*.py
+          files_ignore: |
+            red_queen/games/applications/__init__.py
+            red_queen/games/mapping/__init__.py
+      - name: Install Deps
+        run: |
+          python -m pip install --upgrade pip
+          pip install pytest-sugar
+          pip install .
+      - name: Run every new benchmark
+        run: |
+          for file in ${{ steps.changed-files.outputs.all_changed_and_modified_files }}; do
+            pytest ${file} -n 2 --verbose --tb=no --store
```

Comment:
I feel like we should print the tracebacks in some way. We don't have to use pytest's formatter, but disabling tracebacks with `--tb=no` feels like it would make debugging CI failures harder.

Reply:
I disabled them in order to save some RAM, but it doesn't have an effect on the benchmark runs. Traceback will be enabled in a future commit.